src/houghtransform.jl

```diff
@@ -0,0 +1,102 @@
+
+#function to compute local maximum lines with values > threshold and return a vector containing them
+function findlocalmaxima(accumulator_matrix::Array{Integer,2},threshold::Integer)
```

Comment:
`findlocalmaxima` is already defined in Images.jl. Ideally we would modify `Images.findlocalextrema` to take a function as a first argument, and then you could say `findlocalmaxima(x->x > threshold, accumulator_matrix)`.